### ISSUE TITLE
Bag window does not appear

### ISSUE THREAD

**Issue by Omegashift** [doadin/Baggins#66]
Everything but baggins is disabled. When trying to open the bag window, either by hitting "B" or by clicking on the minimap button, you hear the sound of your bags opening, but the bag window never appears. If you click on the individual bags in your bag bar you can open them manually in the default blizzard view, the backpack does not open at all. Using Baggins-v4.1.0-wrath.zip installed directly via curseforge desktop app. Also tried installing different versions manually from the curseforge website with no luck.

**Comment by jevring** [comment]
Same for me.

**Comment by jevring** [comment]
I get this:
```
1x Baggins\Baggins-Options.lua:2339: bad argument #1 to 'pairs' (table expected, got nil)
[string "=[C]"]: ?
[string "@Baggins\Baggins-Options.lua"]:2339: in function <Baggins\Baggins-Options.lua:2334>
[string "@Baggins\Baggins-Options.lua"]:2431: in function `OnProfileEnable'
[string "@Baggins\Baggins-v4.1.0.lua"]:501: in function <Baggins\Baggins.lua:436>
[string "=[C]"]: ?
[string "@Baggins\Libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:70: in function <...ceBaggins\Libs\AceAddon-3.0\AceAddon-3.0.lua:65>
[string "@Baggins\Libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:527: in function `EnableAddon'
[string "@Baggins\Libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:630: in function <...ceBaggins\Libs\AceAddon-3.0\AceAddon-3.0.lua:615>
```

**Comment by doadin** [comment]
801d9e56cf7d508e8ca62501ed42de8a8e50128f should fix this, this commit is included in v4.1.1 . Please try this version and let me know if you have any other issues.
